python/llm/example/GPU/HuggingFace/Multimodal/qwen2-vl/README.md

```diff
@@ -0,0 +1,151 @@
+# Qwen2-VL
+In this directory, you will find examples on how you could apply IPEX-LLM INT4 optimizations on Qwen-VL models on [Intel GPUs](../../../README.md).  For illustration purposes, we utilize the [Qwen/Qwen2-VL-7B-Instruct](https://huggingface.co/Qwen/Qwen2-VL-7B-Instruct) (or [Qwen/Qwen2-VL-7B-Instruct](https://www.modelscope.cn/models/Qwen/Qwen2-VL-7B-Instruct) for ModelScope) as a reference Qwen2-VL model.
+
+## 0. Requirements
+To run these examples with IPEX-LLM on Intel GPUs, we have some recommended requirements for your machine, please refer to [here](../../../README.md#requirements) for more information.
+
+## Example: Predict Tokens using `generate()` API
+In the example [generate.py](./generate.py), we show a basic use case for a Qwen2-VL model to predict the next N tokens using `generate()` API, with IPEX-LLM INT4 optimizations on Intel GPUs.
+### 1. Install
+#### 1.1 Installation on Linux
+We suggest using conda to manage environment:
+```bash
+conda create -n llm python=3.11
+conda activate llm
+# below command will install intel_extension_for_pytorch==2.1.10+xpu as default
+pip install --pre --upgrade ipex-llm[xpu] --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
+
+pip install transformers==4.45.0 # install transformers which supports Qwen2-VL
+pip install accelerate==0.33.0
+pip install qwen_vl_utils
+pip install "trl<0.12.0"
+
+# [optional] only needed if you would like to use ModelScope as model hub
+pip install modelscope==1.21.0
+pip install addict simplejson python-dateutil sortedcontainers
+```
+
+#### 1.2 Installation on Windows
+We suggest using conda to manage environment:
+```bash
+conda create -n llm python=3.11 libuv
+conda activate llm
+
+# below command will install intel_extension_for_pytorch==2.1.10+xpu as default
+pip install --pre --upgrade ipex-llm[xpu] --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
+
+pip install transformers==4.45.0 # install transformers which supports Qwen2-VL
+pip install accelerate==0.33.0
+pip install qwen_vl_utils
+pip install "trl<0.12.0"
+
+# [optional] only needed if you would like to use ModelScope as model hub
+pip install modelscope==1.21.0
```

Comment:
Refer to https://github.com/modelscope/modelscope/blob/master/requirements/datasets.txt, we may have a try on
`pip install modelscope[datasets]==1.21.0`, as it includes `addict simplejson python-dateutil sortedcontainers`